src/kbmod/run_search.py

```diff
@@ -75,9 +77,7 @@ def do_gpu_search(self, config, search):
 
         search_timer = kb.DebugTimer("Grid Search", debug)
         if debug:
-            print(f"Average Angle = {config['average_angle']}")
-            print(f"Search Angle Limits = {ang_lim}")
-            print(f"Velocity Limits = {config['v_arr']}")
+            print(strategy.debug_string())
```

Comment:
with `str` this becomes `print(strategy)`. Nothing shameful about using dunder methods in Python.